src/redux/sagas/expenditures/fundExpenditure.ts

```diff
@@ -118,8 +117,6 @@ function* fundExpenditure({
       );
     }
 
-    yield put(transactionPending(fundMulticall.id));
```

Comment:
Shouldn't we set the pending transaction also here?
`yield transactionSetPending(fundMulticall.id);`